src/cdk/listbox/listbox.ts

```diff
@@ -40,6 +40,7 @@ import {defer, fromEvent, merge, Observable, Subject} from 'rxjs';
 import {filter, map, startWith, switchMap, takeUntil} from 'rxjs/operators';
 import {ControlValueAccessor, NG_VALUE_ACCESSOR} from '@angular/forms';
 import {Directionality} from '@angular/cdk/bidi';
+import {Platform} from '@angular/cdk/platform';
 
```

Comment:
Can you add a listbox to `src/universal-app/kitchen-sink/kitchen-sink.html`? This is how we test for SSR regressions.